lib/spree/core/mail_interceptor.rb

```diff
@@ -11,7 +11,15 @@ def self.delivering_email(message)
           message.to = Config[:intercept_email]
         end
 
-        message.bcc ||= Config[:mail_bcc] if Config[:mail_bcc].present?
+        if Config[:mail_bcc].present?
+          if message.bcc.present?
+            original_bcc = message.bcc.split(',').map(&:strip)
+            prefered_bcc = Config[:mail_bcc].split(',').map(&:strip)
+            message.bcc = (original_bcc | prefered_bcc).join(", ")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.